package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "acl-sequelize",
-  "version": "0.1.2",
+  "version": "0.1.3",
   "description": "Node Acl Sequelize Backend",
   "main": "index.js",
   "directories": {
     "test": "test"
   },
   "scripts": {
-    "test": "mocha test/runner.js --reporter spec"
+    "test": "mocha test/runner.js --reporter spec --exit"
```

Comment:
This means we tests aren't cleaning up properly.
Probably need to call self.backend.close() after the test ends (Though v4 had issue where connection wasn't closed properly but I think it was already fixed).
Any way if you have the time you can try see if you can clean it properly but let mocha hang for now